### PR TITLE
use SpanBatchSenderFactory instead of MetricBatchSenderFactory

### PR DESCRIPTION
**Pull request by prodjito**

### CODE REVIEW

File: telemetry_examples/src/main/java/com/newrelic/telemetry/examples/SpanExample.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 package com.newrelic.telemetry.examples;
 
 import com.newrelic.telemetry.Attributes;
-import com.newrelic.telemetry.MetricBatchSenderFactory;
+import com.newrelic.telemetry.SpanBatchSenderFactory;
```
Comment (jeffalder) [inline telemetry_examples/src/main/java/com/newrelic/telemetry/examples/SpanExample.java:8]: This needs to move down two lines per google java formatting.

Comment (prodjito) [reply telemetry_examples/src/main/java/com/newrelic/telemetry/examples/SpanExample.java]: Sorry about that! The imports are now fixed.

**Comment by jeffalder** [comment]
@prodjito Let us know if you have any problems with the CLA assistant.

**Comment by jeffalder** [comment]
@prodjito - Sorry to grab this on you, but I decided to implement your changes in #163 so that it was done. Please note that _your contribution_ made us aware of the problem, so thank you! We'd appreciate seeing more of your contributions in our project!

**Pull request closed without merging**